src/internal/version/backup.go

```diff
@@ -39,4 +42,8 @@ const (
 	// OneDriveXLocationRef provides LocationRef information for Exchange,
 	// OneDrive, and SharePoint libraries.
 	OneDrive7LocationRef = 7
+
+	// AllXMigrateUserPNToID marks when we migrated repo refs from the user's
+	// PrincipalName to their ID for stability.
+	AllXMigrateUserPNToID = Backup + 1
```

Comment:
I'm a little on the fence about what the value of this should be. Right now, my understanding is setting it to `Backup + 1` will cause the OD migration code to always create a migration collection but the paths will be the same as selectors is wired to have the UPN for both. If we switch this to `NoBackup` though then we'll have no way to test the migration code :/

I'm not sure if there's a good path forward, do you have any thoughts on this?

Reply:
@ashmrtn see: https://github.com/alcionai/corso/pull/3195

Reply:
3195 is merged